README.md

```diff
@@ -11,8 +9,6 @@ Linux system role to issue and renew SSL certificates.
 Basic usage:
 
 ```yaml
----
```

Comment:
Why remove this part of the example? Isn't it better to have a full playbook?

If this is to be removed, maybe the `vars` part could be added at the role call, rather than a global setting.

Reply:
This was a crazy `sed` overuse, fixed.